homeassistant/components/steam_online/sensor.py

```diff
@@ -164,12 +169,25 @@ def _get_last_online(self):
 
         return None
 
+    def _get_game_image_header(self):
+        """Return the current games header image URL."""
+        if self._game_id is not None:
+            return f"{STEAM_API_URL}{STEAM_APP_ENDPOINT}{self._game_id}/header.jpg"
+
+        return None
```

Comment:
Remove this entire method and move its functionality to `device_state_attributes()`.